.drone.yml

```
@@ -99,13 +99,17 @@ pipeline:
       matrix:
         TESTS: eslint
   vue-build:
-    image: nextcloudci/jsunit:jsunit-5
+    image: nextcloudci/node:node-1
```

```suggestion
    image: nextcloudci/node:node-4
```

That is the name, because of all the testing that was needed.